lib/kue.js

```diff
@@ -445,6 +445,19 @@ Queue.prototype.types = function( fn ) {
 };
 
 /**
+ * Remove the job type.
+ *
+ * @param {String} type
+ * @return {Queue} for chaining
+ * @api public
+ */
+
+Queue.prototype.removeType = function( type ) {
+  this.client.srem(this.client.getKey('job:types'), type);
```

Comment:
According to http://redis.io/commands/srem, If the ZSET is empty, that means the type is not exist in the ZSET. So the process will be ignored. 
If the 'job:types' does not exist,  it is treated as an empty set. So it is the same case as above.
Finally, I think we can assume that 'job:types' is always a set. Am I correct?